test/docker-compose.yaml

```diff
@@ -55,7 +56,7 @@ services:
       - 5432:5432
     environment:
       POSTGRES_USER: postgres
-      POSTGRES_MULTIPLE_DATABASES: keycloak,iam
+      POSTGRES_MULTIPLE_DATABASES: keycloak,iam,logindetails
```

Comment:
Wofür wird die Datenbank "logindetails" gebraucht?

Reply:
Für den Service der dieses Event dann verarbeitet. So muss ich nicht noch ein docker-compose starten sondern kann direkt springboot in meiner IDE laufen lassen.

Reply:
ok, verstanden